Makefile

```diff
@@ -72,6 +73,12 @@ vet: ## Run go vet against code.
 test: manifests fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... -coverprofile cover.out
 
+.PHONY: integration-test
+integration-test: manifests fmt vet envtest ginkgo ## Run integration tests.
```

Comment:
just so we are consistent with the other projects, can we flip this to test-integration please ?

Reply:
Done